src/sentry/static/sentry/app/api.jsx

```diff
@@ -163,6 +163,13 @@ export class Client {
   }
 
   request(path, options = {}) {
+    const hub = Sentry.getCurrentHub();
+    const requestSpan = hub.startSpan({
+      data,
+      op: 'http',
+      description: path,
+    });
```

Comment:
I'm wondering if it's also a good idea to include the `options` object as part of the span payload.

Reply:
The HTTP method would be good to have either in the description or as a data attribute.